node/repo/blockstore_opts.go

```diff
@@ -1,63 +1,12 @@
 package repo
 
 import (
-	"os"
-	"strconv"
-
-	badgerbs "github.com/filecoin-project/lotus/blockstore/badger"
+	versions "github.com/filecoin-project/lotus/blockstore/badger/versions"
 )
 
 // BadgerBlockstoreOptions returns the badger options to apply for the provided
 // domain.
-func BadgerBlockstoreOptions(domain BlockstoreDomain, path string, readonly bool) (badgerbs.Options, error) {
-	opts := badgerbs.DefaultOptions(path)
-
-	// Due to legacy usage of blockstore.Blockstore, over a datastore, all
```

Comment:
Speaking as someone who has struggled with learning about our use of badger these comments are quite valuable and shouldn't be dropped.  Even if some of them are out of date its better to have them around so we know what the original intention was behind these values.  I would copy them over [here](https://github.com/filecoin-project/lotus/pull/12316/files#diff-c8625dbecfeb5154905c055e61a49cd1405c44670fdd05da7f221f9902468e20R157) and maybe on the v4 function too.

Reply:
thanks for this. i've restored the comments and deleted dead code that wasn't being used c4611608f